src/index.html

```diff
@@ -17,6 +17,87 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="navigation">
+      <div class="navigation__wraper">
```

Comment:
I think this div is redundant because you have header tag and you can use it like wrapper